src/main/java/frc/robot/swerve/SwerveConstants.java

```diff
@@ -0,0 +1,138 @@
+                    .withMotionMagicJerk(1);
+    public static final MotionMagicConfigs REAR_LEFT_MOTION_MAGIC_CONFIGS =
+            new MotionMagicConfigs()
+                    .withMotionMagicAcceleration(25000)
+                    .withMotionMagicCruiseVelocity(21288)
+                    .withMotionMagicJerk(1);
+    public static final MotionMagicConfigs REAR_RIGHT_MOTION_MAGIC_CONFIGS =
+            new MotionMagicConfigs()
+                    .withMotionMagicAcceleration(25000)
+                    .withMotionMagicCruiseVelocity(21288)
+                    .withMotionMagicJerk(1);
+
+
+    public static final Slot0Configs[] SLOT_0_ANGLE_CONFIGS = {
+            FRONT_LEFT_ANGLE_PID_GAINS,
+            FRONT_RIGHT_ANGLE_PID_GAINS,
+            REAR_LEFT_ANGLE_PID_GAINS,
+            REAR_RIGHT_ANGLE_PID_GAINS
+    };
+
+    public static final MotionMagicConfigs[] MOTION_MAGIC_CONFIGS = {
+            FRONT_LEFT_MOTION_MAGIC_CONFIGS,
+            FRONT_RIGHT_MOTION_MAGIC_CONFIGS,
+            REAR_LEFT_MOTION_MAGIC_CONFIGS,
+            REAR_RIGHT_MOTION_MAGIC_CONFIGS
+    };
+
+    public static final double MAX_X_Y_VELOCITY = 6380.0 / 60.0 * //[m/s]
+            DRIVE_REDUCTION *
+            WHEEL_DIAMETER * Math.PI;
+
+    public static final double MAX_OMEGA_VELOCITY = MAX_X_Y_VELOCITY / //[m/s]
+            Math.sqrt((ROBOT_LENGTH / 2) * (ROBOT_LENGTH / 2) + (ROBOT_WIDTH / 2) * (ROBOT_WIDTH / 2));
+
+    public static final TalonFXInvertType CLOCKWISE = TalonFXInvertType.Clockwise;
+    public static final TalonFXInvertType COUNTER_CLOCKWISE = TalonFXInvertType.CounterClockwise;
+
+    public static final double NEUTRAL_DEADBAND = 0.15;
+    public static final double XBOX_DEADBAND = 0.15;
+
+
+    public static final double TICKS_PER_RADIAN = FALCON_TICKS / ANGLE_REDUCTION / (Math.PI * 2);
+    public static final double TICKS_PER_METER = (FALCON_TICKS / DRIVE_REDUCTION) / (Math.PI * WHEEL_DIAMETER);
+
+    public static final Translation2d[] wheelPositions = {
+            new Translation2d(ROBOT_LENGTH / 2, ROBOT_WIDTH / 2),   //FL
+            new Translation2d(ROBOT_LENGTH / 2, -ROBOT_WIDTH / 2),   //FR
+            new Translation2d(-ROBOT_LENGTH / 2, ROBOT_WIDTH / 2),  //RL
+            new Translation2d(-ROBOT_LENGTH / 2, -ROBOT_WIDTH / 2)}; //RR
+
+    public static final double MAX_VELOCITY_AUTO = 4.0;
+    public static final double MAX_ACCELERATION_AUTO = 2.5;
+
+    public static double AUTO_X_Kp = 3.20;
+    public static double AUTO_X_Ki = 0.0;
+    public static double AUTO_X_Kd = 0.73;
+    public static double AUTO_Y_Kp = 3.20;
+    public static double AUTO_Y_Ki = 0.0;
+    public static double AUTO_Y_Kd = 0.6;
+    public static double AUTO_ROTATION_Kp = 10;
+    public static double AUTO_ROTATION_Ki = 0.0;
+    public static double AUTO_ROTATION_Kd = 0.0;
+
+    public static double FORWARD_BALANCE_TIME = 0.65;
+    public static double BACKWARD_BALANCE_TIME = 0.55;
```

Comment:
All of these can be deleted, they aren't used.

Reply:
14795a9a765aa85f09e54419d2391772bcb636f3